src/app/api/fetchData.js

```diff
@@ -0,0 +1,13 @@
+import { countriesAxios } from "@/countriesAxios"
+
+const fetchData = async (endpoint, options = {filters: ''}) => {
+    const filtersValues = String(options.filters)
+
+    const result = await countriesAxios.get(`${endpoint}`, {params: { fields: filtersValues}})
```

Comment:
A może tak? Wtedy pozbylibyśmy się dodatkowej zmiennej:

```suggestion
    const result = await countriesAxios.get(endpoint, {params: { fields: filters.join(',')}})
```